src/utils.ts

```diff
@@ -58,9 +59,17 @@ export const thresholdSame = <T>(arr: T[], t: number): T | undefined => {
   return undefined;
 };
 
+export const postWithTraceId = <T>(url: string, data?: Data, options_?: RequestInit, customOptions?: CustomOptions): Promise<T> => {
+  const traceId = crypto.randomUUID();
+  const options: RequestInit = { ...options_, headers: { "x-web3auth-trace-id": traceId } };
+  // eslint-disable-next-line no-console
+  console.log(`Request tracing with traceID=${traceId}`);
```

Comment:
adding this so that it can print out traceID in console.log
why not using `log.info()`? the reason is `log.disableAll();` in loglevel.ts file